### DISCUSSION TITLE
Crazyflie intermittently connecting to Crazyradio

### DISCUSSION THREAD

**Discussion by nbelles** [bitcraze/discussions#341] [Q&A] [answered]
Hello,

I'm currently trying to use the Crazyradio PA to connect to the Crazyflie 2.1. I have been able to connect to it many times before but just the other day when I went to run some of my scripts to connect to the Crazyflie, and it immediately prints `Got link error callback [Too many packets lost] in state [2]` or `Got link error callback [Too many packets lost] in state [1]`. 
I then went back to the basics to make sure my code wasn't what was working incorrectly and tried running one of the examples. When running
`crazyflie-lib-python/examples/radio/scan.py/` (removing the address filter) I get the full output:
```
Scanning interfaces for Crazyflies...
Crazyflies found:
```
I then tried turning the Crazyflie off and running the same example again and again I get the same full output:
```
Scanning interfaces for Crazyflies...
Crazyflies found:
```
I then tried connecting over a USB cable and got the full output:
```
Scanning interfaces for Crazyflies...
Crazyflies found:
usb://0
```
I then tried disconnecting USB and putting the Crazyflie into the boot loader mode (by pressing and holding the power button for 3 seconds on startup) and got the output:
```
Scanning interfaces for Crazyflies...
Crazyflies found:
radio://0/0/2M
```
This has confused me a lot as to why it doesn't recognize the device when running a normal scan but it recognizes the radio when the radio is in boot loader mode. 

Any thoughts? Thanks!

**Comment by tobbeanton** [comment, marked as answer]
I can think of two things. First thing would be that you accidentally changed the radio address (1). (2) is that the FW got corrupt.
(1) Connect to the crazyflie using the USB cable and check the radio configuration settings in cfclient, Connect->Configure 2.X.
(2) Reflash FW

**Comment by nbelles** [comment]
Thank you so much for the insight! Setting the address again seemed to fix the issue. I didn't know that was an option when plugged in over USB using the cfclient.
